README.md

```diff
@@ -99,34 +161,41 @@ After the initial theme setup you can configure `blt/blt.yml` to install and con
 2. Pull Files locally.
 Use BLT to pull all files down from your Cloud environment.
 
-   ```
-   $ blt drupal:sync:files
-   ```
+```
+blt drupal:sync:files
+```
 
 3. Sync the Cloud Database.
 If you have an existing database you can use BLT to pull down the database from your Cloud environment.
-   ```
-   $ blt sync
-   ```
 
+```
+blt sync
+```
 
----
+# Deploy
+
+```
+ACQUIA_CLOUD_REMOTE_GIT=acquia_git_destination blt artifact:deploy --commit-msg "message here" --branch "branch name here" --no-interaction
+```
 
-# Resources 
+# Resources
 
 Additional [BLT documentation](https://docs.acquia.com/blt/) may be useful. You may also access a list of BLT commands by running this:
+
+```
+blt
 ```
-$ blt
-``` 
 
+<!---
```

Comment:
Sounds good, we can add this back when we configure pipelines

Reply:
👍